src/API/timeseries/timeseries.service.js

```diff
@@ -9,7 +9,7 @@ Promise.promisifyAll(MongoClient);
 const debug = Debugger('SH_BE:new-device-handler');
 const STATUS_EVENT = 'status';
 const STRATEGY = 'averages';
-const periods = ['month', 'day', 'hour', 'minute', 'second'];
+const periods = ['month', 'day', 'hour'];
```

Comment:
@kolesnik 
In UI pull request you remove `hour`, but in this backEnd request, you delete `minute` and `second` and left a `hour`, why?